spec/Api/WebClientSpec.php

```diff
@@ -1,92 +1,97 @@
+    const API_ENDPOINT = 'https://api-shipx-pl.easypack24.net/v1';
+    const POINT_NAME = 'AAA666';
+
+    function let(Client $client): void
+    {
+        $this->beConstructedWith($client);
+    }
+
+    function it_is_initializable(): void
+    {
+        $this->shouldHaveType(WebClient::class);
+        $this->shouldImplement(WebClientInterface::class);
+    }
+
+    function it_creates_api_endpoint_for_shipment(
+        ShippingGatewayInterface $shippingGateway,
+        OrderInterface $order,
+        ShipmentInterface $shipment,
+        PaymentInterface $payment,
+        PaymentMethod $paymentMethod,
+        AddressInterface $shippingAddress,
+        CustomerInterface $customer
+    ): void {
+        $shippingGateway->getConfigValue('service')->willReturn('inpost_locker_standard');
+        $shippingGateway->getConfigValue('is_return')->willReturn(false);
+        $shippingGateway->getConfigValue('insurance_amount')->willReturn(null);
+        $shippingGateway->getConfigValue('additional_services')->willReturn([]);
+        $shippingGateway->getConfigValue('cod_payment_method_code')->willReturn(null);
+        $shippingGateway->getConfigValue('environment')->willReturn('sandbox');
+        $shippingGateway->getConfigValue('access_token')->willReturn('1234567890.abcdefghij');
+        $shippingGateway->getConfigValue('organization_id')->willReturn(self::ORGANIZATION_ID);
+        $shippingGateway->getConfigValue('environment')->willReturn('https://sandbox-api-shipx-pl.easypack24.net');
+
+        $customer->getId()->willReturn(1);
+        $customer->getEmail()->willReturn('email@email.com');
+
+        $shipment->getOrder()->willReturn($order);
+        $shipment->getShippingWeight()->willReturn(1.0);
+        $shipment->getId()->willReturn(1);
+
+        $order->getCustomer()->willReturn($customer);
+        $order->getShippingAddress()->willReturn($shippingAddress);
+        $order->getNumber()->willReturn(12);
+        $order->getCurrencyCode()->willReturn('USD');
+        $order->getTotal()->willReturn(100);
+        $order->getPayments()->willReturn(new ArrayCollection([$payment->getWrappedObject()]));
+        $order->getNotes()->willReturn('Notes');
+
+        $payment->getMethod()->willReturn($paymentMethod);
+
+        $shippingAddress->getCompany()->willReturn('Company name');
+        $shippingAddress->getFirstName()->willReturn('First name');
+        $shippingAddress->getLastName()->willReturn('Last name');
+        $shippingAddress->getPhoneNumber()->willReturn('666666666');
+        $shippingAddress->getStreet()->willReturn('Street 666');
+        $shippingAddress->getCity()->willReturn('City');
+        $shippingAddress->getPostcode()->willReturn('66-666');
+        $shippingAddress->getCountryCode()->willReturn('PL');
+
+        $this->setShippingGateway($shippingGateway);
+
+        $this->getApiEndpointForShipment()->shouldReturn(self::API_ENDPOINT . '/organizations/' . self::ORGANIZATION_ID .'/shipments');
```

Comment:
It would be good to prepare few tests with single assertions.
Create a private method(s) to set up the fixtures for tests